service-base/src/main/java/org/eclipse/hono/service/plan/PrometheusBasedResourceLimitChecks.java

```diff
@@ -172,13 +249,14 @@ public PrometheusBasedResourceLimitChecks(final WebClient webClient,
                     .orElseGet(() -> executeQuery(queryParams)
                             .map(bytesConsumed -> addToCache(limitsCache, key, bytesConsumed)))
                     .map(bytesConsumed -> {
-                        if ((bytesConsumed + payloadSize) <= maxBytes) {
+                        if ((bytesConsumed + payloadSize) <= allowedMaxBytes) {
                             return Boolean.FALSE;
                         } else {
-                            log.trace("data limit exceeded for tenant. [{}] are [bytesUsed:{}, {}:{}, {}:{}, {}:{}]",
-                                    tenant.getTenantId(), bytesConsumed, TenantConstants.FIELD_MAX_BYTES,
-                                    maxBytes,
+                            log.trace(
+                                    "data limit exceeded for tenant. [{}] are [bytesUsed:{}, allowed max-bytes:{}, {}:{}, {}:{}, {}:{}]",
```

Comment:
this is no english sentence

Reply:
That was a typing mistake, I will update it.